src/radical/pilot/agent/scheduler/continuous.py

```diff
@@ -544,13 +563,16 @@ def _alloc_mpi(self, unit):
         requested_procs  = cud['cpu_processes']
         threads_per_proc = cud['cpu_threads']
         requested_gpus   = cud['gpu_processes']
+        gpu_chunkg       = cud['gpu_threads']
         lfs_per_process  = cud['lfs_per_process']
         mem_per_process  = cud['mem_per_process']
 
         # make sure that processes are at least single-threaded
         if not threads_per_proc:
             threads_per_proc = 1
 
+        if not gpu_chunk:  gpu_chunk  = 1
```

Comment:
There is another PR that introduces this, right?

Reply:
Yes, some PRs are overlapping by now.  That will be resolved when merging back from devel though...